solution/solution.py

```diff
@@ -29,11 +29,47 @@ def _multiply_by_eight(inputs=None):
     #     x = x + x  # x = input * 2^3 = input * 8
     return [8*i for i in inputs]
 
+def _basic_if(inputs=None):
```

Comment:
`basic_if` and `else_if` are pretty much exactly the same question.

Reply:
Agreed. I will think about better way. Also that's one of reasons why I used the primitive block (having only if-do) to make 1st and 2nd game different. Please let me know.

Reply:
I re-wrote 1st and 2nd games based on your comments. Thanks because it was very good exercise for me to practice Blockly games.